base/docs/Docs.jl

```diff
@@ -66,6 +66,7 @@ using .Base: RefValue, mapany
 import ..CoreDocs: lazy_iterpolate
 
 export doc
+public hasdoc
```

Comment:
```suggestion
public hasdoc, undocumented_names
```

Reply:
I was holding off until hearing from Triage about whether `undocumented_names` belongs in `Docs` vs `Test`/`Aqua` as in the comment in that thread. Please comment there if you have a view on that.